torch_geometric/utils/loop.py

```diff
@@ -201,6 +201,7 @@ def add_self_loops(
     is_sparse = is_torch_sparse_tensor(edge_index)
     if is_sparse:
         assert edge_attr is None
+        size = edge_index.size()
```

Comment:
```suggestion
        size = (edge_index.size(0), edge_index.size(1))
```
seems to be needed for TorchScript.